solutions/cf-231A.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: cf-231A
+source: CF
+title: Team
+author: Sathvik Chundru, Kevin Sheng
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/5486)
+
+## Explanation
+
+The simplest solution is to input three integers per row and check 
+if their sum is greater than or equal to two. 
+
+## Implementation
+
```

Comment:
add time complexity

Reply:
bro i suck at calculating time and idk latex can u calculate and add it pls